<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>2</width>
    <height>2</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>2</width>
    <height>2</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MTP Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="progresswidget" native="true">
      <property name="minimumSize">
       <size>
        <width>855</width>
        <height>27</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="progressbar">
       <item>
        <widget class="QLabel" name="operation">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cancel_operation">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>320</height>
         </size>
        </property>
        <property name="title">
         <string>Search Devices</string>
        </property>
        <widget class="QTreeView" name="devinfo">
         <property name="geometry">
          <rect>
           <x>17</x>
           <y>90</y>
           <width>191</width>
           <height>231</height>
          </rect>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>78</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Device Info:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="connect">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>60</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>connect</string>
         </property>
        </widget>
        <widget class="QPushButton" name="refresh">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>31</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="DevicesList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>171</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="devicetabs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="foldervt">
         <attribute name="title">
          <string>File Manager</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QSplitter" name="splitter">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QTreeView" name="devfilestree">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>261</height>
                 </size>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
                </property>
                <property name="dragEnabled">
                 <bool>true</bool>
                </property>
                <property name="dragDropOverwriteMode">
                 <bool>true</bool>
                </property>
                <property name="dragDropMode">
                 <enum>QAbstractItemView::DragDrop</enum>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
               </widget>
               <widget class="QWidget" name="widget_2" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>400</width>
                  <height>262</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="widget" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>301</width>
                     <height>41</height>
                    </size>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QToolButton" name="adress_back">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="adress_parent">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="adress_refresh">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="adressline"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="DevFilesView" name="devfilesview">
                   <property name="minimumSize">
                    <size>
                     <width>301</width>
                     <height>211</height>
                    </size>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::CustomContextMenu</enum>
                   </property>
                   <property name="acceptDrops">
                    <bool>true</bool>
                   </property>
                   <property name="editTriggers">
                    <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
                   </property>
                   <property name="dragDropOverwriteMode">
                    <bool>true</bool>
                   </property>
                   <property name="dragDropMode">
                    <enum>QAbstractItemView::DragDrop</enum>
                   </property>
                   <property name="defaultDropAction">
                    <enum>Qt::CopyAction</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mediavt">
         <attribute name="title">
          <string>Media</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DevFilesView</class>
   <extends>QTreeView</extends>
   <header>devfilesview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
